content/en/agent/configuration/dual-shipping.md

```diff
@@ -141,6 +141,20 @@ clusterAgent:
     - name: DD_ADDITIONAL_ENDPOINTS
       value: '{"https://app.datadoghq.com": ["apikey2"]}'
 ```
+### Cluster Agent metrics provider
+
+To ensure autoscaling is resilient to failure, configure the Cluster Agent to selectively fetch any metrics you use for HPA (both Kubernetes and standard/custom application metrics) from multiple Datadog regions. Configure the Datadog Cluster Agent manifest with several endpoints:
```

Comment:
```suggestion
To ensure autoscaling is resilient to failure, configure the Cluster Agent to selectively fetch any metrics you use for HPA (both Kubernetes and standard or custom application metrics) from multiple Datadog regions. Configure the Datadog Cluster Agent manifest with several endpoints:
```

Reply:
I'm not totally clear on what `standard` means in this case -- is that any default Datadog or integration metric, or is it a type of application metric?

Reply:
```suggestion
To ensure autoscaling is resilient to failure, configure the Cluster Agent to run your metric queries for the HPA against your multiple Datadog regions with dual-shipped data. Configure the Datadog Cluster Agent manifest with several endpoints:
```